public/components/visualizations/visualization.tsx

```diff
@@ -29,7 +29,7 @@ export const Visualization = ({
     // Markdown, it does not depend on if there is data
     if (vis.id === VIS_CHART_TYPES.Text) return [true, ''];
 
-    if (isEmpty(series)) return [false, VISUALIZATION_ERROR.INVALID_DATA]; // series is required to any visualization type
```

Comment:
If the query is not valid and returns no data, what would it show?

Reply:
I assumed that an empty series would mean that data hasn't been put in yet. Shouldn't this be the case? For invalid queries there are other checks in this file that will display invalid data if invalid.

Reply:
we can check with UX team on this